Jint.Tests.Test262/Test262Test.cs

```diff
@@ -323,18 +323,86 @@ public static IEnumerable<object[]> SourceFiles(string pathPrefix, bool skipped)
                     reason = "Esprima problem, Unexpected token *";
                 }
 
+                if (name.StartsWith("language/white-space/mongolian-vowel-separator-eval.js"))
```

Comment:
if it's a full file name it can also be in skipped.json , there's own section for esprima problems